### DISCUSSION TITLE
Modelviewer can't detect floor and the object floats on Android

### DISCUSSION THREAD

**Discussion by ahmetbugrayasar** [google/model-viewer#2247] [Q&A]
### Description
Our glb models both large and small, all float. We've tried Chrome and Safari. On iOS the project works flawlessly. However, on Android, it floats.
Phones we tried were a Huawei Mate10 and an LG G8.

#### Live Demo
https://ersincetin.tk/
Here is the live Demo

### Version
https://unpkg.com/@google/model-viewer/dist/model-viewer.js
- model-viewer: vX.X.X

### Browser Affected

- [X] Chrome, version: 89.0.4389.90
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [x] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [x] WebXR
- [x] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
iOS uses USDZ, while the others use the glTF file, so the error is the conversion between the two and how their coordinate systems are represented. If you edit the glTF directly you can likely find and remove the offending offset.

**Comment by elalish** [comment]
Wait, I just saw your title says the floor is not found. And you're sure you've tried this in both SceneViewer and WebXR modes? Can you post screen recordings so I can see what you are seeing?

**Comment by ahmetbugrayasar** [comment]
![gltfissue1](https://user-images.githubusercontent.com/59139830/112876639-6b223a00-90ce-11eb-9daf-c92b5649b1ac.jpg)

https://user-images.githubusercontent.com/59139830/112876654-707f8480-90ce-11eb-9ed1-0e4df04aef21.mp4


https://user-images.githubusercontent.com/59139830/112876673-74130b80-90ce-11eb-81b5-5d99e1a09d54.mp4

I didn't Include the iOS cases because they already work flawlessly. First vid is SceneViewer prioritized and second one is WebXR prioritized.

**Comment by elalish** [comment]
It looks like the scale of your model is off (way too large). I believe the formats use different units, so this is a common problem in conversions. glTF is always meters, so check your bounding box. 

Also keep in mind that to find the floor you need to move your phone side to side, not just rotate it.
